DataGrid: Update remoteOperations (#7851)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDataGrid/1 Configuration/remoteOperations/remoteOperations.md

---
---
##### shortDescription
Specifies DataGrid operations to be executed on the server side.

---

> Server-side data processing improves component performance on large datasets. The following table lists **remoteOperation** configuration values and what operations each value affects:

Reviewer: ```suggestion
Server-side data processing improves component performance with large datasets. The following table lists **remoteOperation** configuration values and which operations each value affects:
```
